AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -513,9 +513,7 @@ - (void)checkLatestRelease:(NSString *)updateToken distributionGroupId:(NSString
                                       @"3. The app is running in a non-adhoc environment. "
                                       @"Detach the debugger and restart the app and/or run the app with the release configuration "
                                       @"to enable the feature.");
-    @synchronized(self) {
```

Comment:
I saw a few places in this file we are surrounding `self.updateFlowInProgress` with synchronized block. With `updateFlowInProgress` changing to atomic, do we still need synchronized block in those places?

Reply:
I'm wondering about race conditions as well.

Reply:
We don't have any operations in the synchronized block other than setting it. It doesn't change anything.

Reply:
It's not marked to be tested in app. Can you check?